client/src/test/java/io/pravega/client/stream/impl/EventPointerTest.java

```diff
@@ -45,6 +45,7 @@ public void testEventPointerImpl() throws IOException, ClassNotFoundException {
 
         name.append("/");
         name.append(segmentId);
+        name.append("#secondary.0");
```

Comment:
What is the point of "secondary" doesn't '#' cover it?

Reply:
answered here https://github.com/pravega/pravega/pull/2600#issuecomment-391576540